rkt/profile_stub.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !profile
+
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func parseProfileFlags(cmdRkt *cobra.Command) {}
+
+func startProfile() (cpufile *os.File, memfile *os.File, err error) {
+	return nil, nil, nil
```

Comment:
I suggest to write out a log entry for the user as a hint, that profiling was disabled at compile time.